### PR TITLE
Tweak documentation style a bit

### PR DESCRIPTION
**Pull request by h-east**
The following tweaks were made:

- Extra whitespace (at the end of a line)
- Whitespaces where a tab should be
- Over 80 columns
- Not a two-space convention
- etc...

### CODE REVIEW

File: runtime/doc/if_pyth.txt [hunk 1]
```diff
@@ -201,7 +201,7 @@ vim.eval(str)						*python-eval*
 	[{'cmd': '/^eval_expr(arg, nextcmd)$/', 'static': 0, 'name': ~
 	'eval_expr', 'kind': 'f', 'filename': './src/eval.c'}] ~
 
-	NOTE: In vim9script, local variables in def functions are not visible
+	NOTE: In Vim9 script, local variables in def functions are not visible
 	to to python evaluations. To pass local variables to python evaluations,
```
Comment (yegappan) [inline runtime/doc/if_pyth.txt]: Can you also remove the extra "to"  here?

Comment (h-east) [reply runtime/doc/if_pyth.txt]: Fixed. Thanks for letting me know.

**Comment by chrisbra** [comment]
thanks!

**Comment by chrisbra** [comment]
included as of 624bb83619cbd685b1902b016ca

**Pull request closed without merging**